### ISSUE TITLE
Replacing service in DI  from functional test not working

### ISSUE THREAD

**Issue by denkurbatov** [Codeception/module-phalcon4#3]
### Expected and Actual Behavior

I'm trying replace service in DI when write functional test. I'm need it for mock services in DI.

Simple example:

In services.php file:

```php
$di->set('myService', Elements::class);
```

In IndexCept.php:

```php
$I->addServiceToContainer('myService', NotFoundPlugin::class);
```

In IndexController.php, indexAction:

```php
var_dump($this->getDI()->get('myService'));
exit;
```

I expect that var_dump() function return object of NotFoundPlugin class, but get Elements class.

I'm just trying different approaches, like get application instance, get default DI, etc. Still not working.
Thank you for any help or suggestions.

### Details

* Phalcon Framework version: 3.4.1
* Codeception version: 2.5.6
* Phalcon Demo Application version: master
* PHP Version: 7.1
* Operating System: Ubuntu
* Server: Nginx

**Comment by ruudboon** [comment]
@denkurbatov Do you have the possibility to test with Phalcon 4? It's fixed in v1.0.2

**Comment by denkurbatov** [comment]
@ruudboon Yes, I will try. But I'm not sure that my old apps ready for Phalcon 4 :)  Will let you know, thank you a lot.

**Comment by denkurbatov** [comment]
@ruudboon I have checked it, and it works as expected with Phalcon 4 and this codeception module. Thank you a lot for a quick fix.

Is it possible to make same fix to module https://github.com/Codeception/module-phalcon as well? I already checked, same fix works also in old module version :) Thank you anyway, even if you don't support old version of Phalcon anymore will do my best for updating old apps to Phalcon 4 :)
